CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 CMAKE_MINIMUM_REQUIRED( VERSION 3.3 )
 
-PROJECT( FCalClusterer CXX )
+PROJECT( FCalClusterer CXX C)
```

Comment:
What is the warning or error message that makes it require "C" ?

Reply:
```
-- Looking for include file pthread.h
CMake Error at /data/ilcsoft/CMake/3.3.2/share/cmake-3.3/Modules/CheckIncludeFiles.cmake:79 (try_compile):
  Unknown extension ".c" for file

    /data/ilcsoft/HEAD/FCalClusterer/HEAD/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c

  try_compile() works only for enabled languages.  Currently these are:

    CXX

  See project() command to enable other languages.
```

Reply:
OK. Is this on mac? I don't see it with cmake 3.6.2 on linux

Reply:
Yes, this is on my macbook ...